package.json

```diff
@@ -67,6 +67,7 @@
     "babel-jest": "^29.7.0",
     "babel-loader": "^9.1.3",
     "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
What was the symptom that using this resolves?

Reply:
The date locale priority uses `navigator.language` to get the locale setting. In order for `navigator` to exist, those tests needed to be run in the jsdom environment.